ntrans_gui.py

```diff
@@ -202,17 +243,16 @@ def generate_ntrans_dictionary(self) -> None:
 
         for key in user_choices:
             if not user_choices[key]:
-                print(
-                    "All fields need to be filled in before generating the N-Trans dictionary."
-                )  # TODO: Change print to label
-                break  # TODO: break needs to change to `return`. Will be fixed in #23
+                self.flash_warning_label()
+                return
 
         # TODO: Check filepath for validity
         if self.thread is None:
             self.progress_queue: queue.Queue[int] = queue.Queue()
             # Calls logic in ntrans.py
             self.thread = threading.Thread(
-                target=ntrans.read_ngram_files, args=[user_choices, self.progress_queue]
+                target=ntrans.read_ngram_files,
+                args=[user_choices, self.progress_queue],
```

Comment:
The code hasn't changed even though black formatting changed. This is a black weirdness: it accepts either of the two formats, and when auto-formatting, it chooses the format based on whether there's a trailing `,` after the last argument. If you want to disable this, like I did in porcupine, create `pyproject.toml` with this content:

```toml
[tool.black]
skip-magic-trailing-comma = true
```